docs(website): add databases overview page and update links (#1691)
aerogear/graphback: docs/databases/introduction.md

---
id: overview
title: Overview
sidebar_label: Overiew 
---

At the core of a CRUD API is the database that Graphback interacts with by providing a rich set of tools and APIs that help you get your database working with your GraphQL application in minutes.

> Graphback has a [CRUD abstraction](../crud/overview) with a number of out-of-the-box implementations for [PostgreSQL](postgres.md), [MongoDB](mongodb.md) and [SQLite](sqlite.md) databases.

Author: ```
Graphback has a [CRUD abstraction](../crud/overview) with a number of out-of-the-box implementations for [PostgreSQL](postgres.md), [MongoDB](mongodb.md) databases.  
[SQLite](sqlite.md) database is also partially supported and we do not recommend using it in production environment, however it is a good choice for testing or development purposes. 
```

WDYT?

> Additionally, Graphback integrates with [`graphql-migrations`](../graphql-migrations/intro.md) so that you can migrate your databases using your business models defined in the GraphQL schema. 

Reviewer: ```suggestion
For PostgreSQL, Graphback integrates with [`graphql-migrations`](../graphql-migrations/intro.md) so that you can migrate your databases using your business models defined in the GraphQL schema. 
```

Reviewer: Ah.. Ignore me

> Additionally, Graphback integrates with [Graphql Migrations`](../graphql-migrations/intro.md) so that you can migrate your databases using your business models defined in the GraphQL schema. 

Reviewer 2: ```suggestion
Additionally, Graphback integrates with [Graphql Migrations`](../graphql-migrations/intro.md) so that you can migrate your databases using your business models defined in the GraphQL schema. 
```
```suggestion
Additionally, Graphback integrates with [GraphQL Migrations`](../graphql-migrations/intro.md) so that you can migrate your databases using your business models defined in the GraphQL schema. 
```

> Additionally, Graphback integrates with [GraphQL Migrations`](../graphql-migrations/intro.md) so that you can migrate your databases using your business models defined in the GraphQL schema. 

Reviewer 2: ```suggestion
Additionally, Graphback integrates with [GraphQL Migrations](../graphql-migrations/intro.md) so that you can migrate your databases using your business models defined in the GraphQL schema. 
```

:::note
GraphQL Migrations only operates on relational databases e.g PostgreSQL and does not work with MongoDB as shown in the [compatibility](#compatibility) table below.
:::

## Compatibility

> | Operation | PostgreSQL | MongoDB | SQLite |

Author: I think we should keep it as it is a good choice for testing / dev envs. So showing how far we do support it is good.
